File: FCTSeek-frontend-movil/src/services/valoracionService.ts
// src/services/valoracionService.ts
import { api } from '../config/api';

export interface ValoracionRequest {
  empresaId: number;
  puntuacion: number;
}

export interface ValoracionResponse {
  id: number;
  empresaId: number;
  usuarioId: number;
  puntuacion: number;
  rolValorador: string;
  anioAcademico?: string;
}

export const valoracionService = {
  getByEmpresa: (empresaId: number) =>
    api.get<ValoracionResponse[]>(`/valoraciones/empresa/${empresaId}`),

  crear: (data: ValoracionRequest) =>
    api.post<ValoracionResponse>('/valoraciones', data),
};

File: FCTSeek-desktop/src/main/java/com/practicas/service/ValoracionService.java
package com.practicas.service;

import com.practicas.util.ApiClient;
import com.practicas.util.ApiClient.ApiException;

import java.util.Map;

public class ValoracionService {

    public static void crear(long empresaId, int puntuacion) throws ApiException {
        ApiClient.postRaw("/valoraciones", Map.of("empresaId", empresaId, "puntuacion", puntuacion));
    }
}
